src/app/view/util/svg.mouse.controller.ts

```diff
@@ -35,12 +35,21 @@ export class SVGMouseController {
   private viewboxIsFixed = false;
 
   private lastMouseEventTimeStamp: number = undefined;
+  private static ctrlKeyPressed = false;
```

Comment:
This is a static variable, but the callbacks that update it are per-instance. Maybe this should be per-instance?